NAMESPACE

```diff
@@ -0,0 +1,4 @@
+# Generated by roxygen2: do not edit by hand
```

Comment:
I was curious whether I should add his file and also files in `man`. From my point of view, they are autogenerated and need not to be kept under version control. But then R-CMD tests fails. What will happen if I update docstrings in R files, but will not regenerate `NAMESPACE` and files in `man`? I hope it will also fail. 

Coming from GAP, I'd consider the workflow when documentation is autogenerated when a release is wrapped, and also is autogenerated in GitHub actions script before running R-CMD check, but not kept under version control. But it seems from https://github.com/r-lib/actions that in the R community it's done differently, so I am fine with keeping these files. I will experiment to see what happens with CI if I will skip updating them, modifying docstrings in R files.

Reply:
@alex-konovalov I don't think I can set up the code coverage as I will not have access to the repo from the https://codecov.io/gh side to set it up. I got dssd code coverage working however so I think you can just copy the coverage yaml file across from there once you have added this library to the codecov site. 

As people can install packages from github directly the documentation will need to exist already in the man folder. The process of generating the documentation and NAMESPACE is a user initiated process carried out by an R library (which assists in documentation writing) this is not an internal process carried out automatically when packages are built. CRAN checks involve ensuring the documentation matches the code so they will throw errors if the documentation is not kept up-to-date.

Reply:
@LHMarshall I am not sure that you need to activate anything at Codecov - it might be that it will appear as soon as the `codecov.yml` file will appear (?). So, shall I merge this first, and then would you try the next PR to copy your dssd setup?

Reply:
@alex-konovalov yes merge this then I will try adding the coverage stuff then.

Reply:
@LHMarshall thanks - please go ahead then!